lib/ia2/acacia_ia2.i

```diff
@@ -21,83 +22,83 @@ namespace std {
-%catches(std::runtime_error) IATable2::getNColumns();
-%catches(std::runtime_error) IATable2::getNRows();
-%catches(std::runtime_error) IATable2::toString();
+%catches(std::runtime_error) acacia::IATable2::isNull();
+%catches(std::runtime_error) acacia::IATable2::getNColumns();
+%catches(std::runtime_error) acacia::IATable2::getNRows();
+%catches(std::runtime_error) acacia::IATable2::toString();
 
-%catches(std::runtime_error) IATableCell::isNull();
-%catches(std::runtime_error) IATableCell::getColumnExtent();
-%catches(std::runtime_error) IATableCell::getColumnIndex();
-%catches(std::runtime_error) IATableCell::getRowExtent();
-%catches(std::runtime_error) IATableCell::getRowIndex();
-%catches(std::runtime_error) IATableCell::toString();
+%catches(std::runtime_error) acacia::IATableCell::isNull();
+%catches(std::runtime_error) acacia::IATableCell::getColumnExtent();
+%catches(std::runtime_error) acacia::IATableCell::getColumnIndex();
+%catches(std::runtime_error) acacia::IATableCell::getRowExtent();
+%catches(std::runtime_error) acacia::IATableCell::getRowIndex();
+%catches(std::runtime_error) acacia::IATableCell::toString();
 
-%catches(std::runtime_error) IAText::isNull();
-%catches(std::runtime_error) IAText::getCaretOffset();
-%catches(std::runtime_error) IAText::getNCharacters();
-%catches(std::runtime_error) IAText::getText(long start_offset, long end_offset);
-%catches(std::runtime_error) IAText::toString();
+%catches(std::runtime_error) acacia::IAText::isNull();
+%catches(std::runtime_error) acacia::IAText::getCaretOffset();
+%catches(std::runtime_error) acacia::IAText::getNCharacters();
+%catches(std::runtime_error) acacia::IAText::getText(long start_offset, long end_offset);
+%catches(std::runtime_error) acacia::IAText::toString();
 
-%catches(std::runtime_error) IAValue::isNull();
-%catches(std::runtime_error) IAValue::getCurrentValue();
-%catches(std::runtime_error) IAValue::getMaximumValue();
-%catches(std::runtime_error) IAValue::getMinimumValue();
-%catches(std::runtime_error) IAValue::toString();
+%catches(std::runtime_error) acacia::IAValue::isNull();
+%catches(std::runtime_error) acacia::IAValue::getCurrentValue();
+%catches(std::runtime_error) acacia::IAValue::getMaximumValue();
+%catches(std::runtime_error) acacia::IAValue::getMinimumValue();
+%catches(std::runtime_error) acacia::IAValue::toString();
 
 #define ACACIA_EXPORT
 %include <acacia/ia2/ia_2.h>
 %include <acacia/ia2/ia_action.h>
 %include <acacia/ia2/ia_component.h>
 %include <acacia/ia2/ia_hyperlink.h>
-%include <acacia/ia2/ia_node.h>
 %include <acacia/ia2/ia_table2.h>
 %include <acacia/ia2/ia_table_cell.h>
 %include <acacia/ia2/ia_text.h>
 %include <acacia/ia2/ia_value.h>
+%include <acacia/ia2/ia_node.h>
 
-IANode findRootIANodeForName(const std::string& name, const int pid = 0);
-IANode findRootIANodeForPID(const int pid);
+%rename(findRootIANodeForPID) acacia::findRootIANodeForPID;
+%rename(findRootIANodeForName) acacia::findRootIANodeForName;
```

Comment:
hmm do we need this? I think we don't!

Reply:
I can't build on Windows right now, but I'll check this later once I get it working again :)